app/helpers/hosts_helper.rb

```diff
@@ -261,6 +261,7 @@ def overview_fields(host)
     fields += [[_("Puppet Environment"), (link_to(host.environment, hosts_path(:search => "environment = #{host.environment}")))]] if host.environment.present?
     fields += [[_("Host Architecture"), (link_to(host.arch, hosts_path(:search => "architecture = #{host.arch}")))]] if host.arch.present?
     fields += [[_("Operating System"), (link_to(host.operatingsystem.to_label, hosts_path(:search => "os_description = #{host.operatingsystem.description}")))]] if host.operatingsystem.present?
+    fields += [[_("PXE Loader"), host.operatingsystem.pxe_loader_name(host)]] if host.operatingsystem.present?
```

Comment:
What about hiding this when the OperatingSystem does not support any PXE loaders.